### ISSUE TITLE
Is Scintilla_ahk2 also available in V2 code

### ISSUE THREAD

**Issue by dmtr99** [TheArkive/scintilla_ahk2#1]
It would be cool if I could use this highlighter for the V2 convertor, is this library also available in v2?

**Comment by TheArkive** [comment]
Sorry I missed this.  This lib is for v2.  I'll have to double check to make sure the example is still working.

Have you tried this in v2?

**Comment by dmtr99** [comment]
Great, it seems to be working well now.

**Comment by TheArkive** [comment]
Ah, I actually didn't change anything.  I only separated the example from the lib as lexikos suggested in another issue.

I did however add the `Scintilla.dll` file just for convenience, but I may remove it again in the future.  I mention this in the README, to download the scintilla DLL.

I'm guessing that may have been the "issue" you were dealing with when you opened this?  The DLL was meant to be downloaded separately at the time.  Either that or you downloaded a DLL that was updated and there were compatibility issues.

As it stands now this lib is out of date.  I plan to add new API stuff from the new Scintilla dll eventually.
